pysts/clasrel_layers.py

```diff
@@ -0,0 +1,133 @@
+from keras.layers.wrappers import TimeDistributed as td
+
+from keras import activations, initializations
+import keras.constraints
+import keras.regularizers
+from keras.layers.core import MaskedLayer, Layer, TimeDistributedDense, TimeDistributedMerge, Activation
+import keras.backend as K
+import theano.tensor as T
+import numpy as np
+
+def relu(x):
+    return K.switch(x > 0, x + 0.01, 0.01)
```

Comment:
Before the introduction of stabilizing constant in the clasrel denominator, this was a necessary relu adjustment, now its outdated.